src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray_1.Builder.cs

```diff
@@ -613,21 +613,7 @@ public int LastIndexOf(T item, int startIndex, int count, IEqualityComparer<T> e
             /// </summary>
             public void Reverse()
             {
-                // The non-generic Array.Reverse is not used because it does not perform
-                // well for non-primitive value types.
-                // If/when a generic Array.Reverse<T> becomes available, the below code
-                // can be deleted and replaced with a call to Array.Reverse<T>.
```

Comment:
System.Collections.Immutable has a build flavor that targets older netstandard versions. We are not building it right now, but @ericstj is working on getting it fixed. This change will need to be under netcoreapp ifdef here.